src/App/App.js

```diff
@@ -13,11 +15,14 @@ function App() {
         <Route path="/game">
           <UserStats />
           <Route path="/game/map" component={Map} />
+          <Route path="/game/:location/encounter" component={Encounters}/>
           <aside>
             <Description />
             <Actions />
           </aside>
         </Route>
+        <Route exact="/" component={Login}>
```

Comment:
So is this going to be our homepage/startpage

Reply:
Correct, we should theoretically change it, but this is rather standard from my limited experience.